rust_src/src/keymap.rs

```diff
@@ -773,4 +772,50 @@ pub fn apropos_internal(regexp: LispStringRef, predicate: LispObject) -> LispObj
     }
 }
 
+// This function may GC (it calls Fkey_binding).
+/// Return the remapping for command COMMAND.
+/// Returns nil if COMMAND is not remapped (or not a symbol).
+///
+/// If the optional argument POSITION is non-nil, it specifies a mouse
+/// position as returned by `event-start' and `event-end', and the
+/// remapping occurs in the keymaps associated with it.  It can also be a
+/// number or marker, in which case the keymap properties at the specified
+/// buffer position instead of point are used.  The KEYMAPS argument is
+/// ignored if POSITION is non-nil.
+///
+/// If the optional argument KEYMAPS is non-nil, it should be a list of
+/// keymaps to search for command remapping.  Otherwise, search for the
+/// remapping in all currently active keymaps.
+#[lisp_fn(min = "1")]
+pub fn command_remapping(
+    command: Option<LispSymbolRef>,
```

Comment:
It's quite strange that the original function behaves this way, but the original function accepts *any* type and returns Qnil if it's not a symbol. I recommend changing `command` to a `LispObject` and then doing `let c = match command.as_symbol() ...`